app/jobs/sync_with_blue_alliance_job.rb

```diff
@@ -16,25 +17,58 @@ def team_list(event_key)
 		response.parsed_response
 	end
 	
-	def perform()
+	def match_list(event_key)
+		response = self.class.get("/event/#{event_key}/matches/simple")
+		response.parsed_response
+	end
+	
+	def get_team(team_key)
+		response = self.class.get("/team/#{team_key}/simple")
+		response.parsed_response
+	end
+	
+	def get_competition(competition_key)
+		response = self.class.get("/event/#{competition_key}/simple")
+		response.parsed_response
+	end
+	
+	def perform(team_number)
```

Comment:
It would make this code more readable and maintainable if you broke this up into smaller helper functions.

Reply:
I'll open an issue for that